### ISSUE TITLE
Manjaro Gnome v40.5 error

### ISSUE THREAD

**Issue by Tealk** [alexduf/gnome-github-notifications#38]
I get still the same error like #32 on Manjaro Gnome v40.5

The settings of extension github.notifications@alexandre.dufournet.gmail.com had an error:
```
TypeError: hideWidgetBox.pack_start is not a function
```

Stack trace:
```
buildPrefsWidget@/home/tealk/.local/share/gnome-shell/extensions/github.notifications@alexandre.dufournet.gmail.com/prefs.js:28:17
_init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:209:40
OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:115:22
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
run@resource:///org/gnome/gjs/modules/script/package.js:206:19
start@resource:///org/gnome/gjs/modules/script/package.js:190:8
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

**Comment by goldyfruit** [comment]
Same on Pop!_OS 21.10

**Comment by machitgarha** [comment]
This should be closed with #33.
